pkg/converter/types.go

```diff
@@ -79,6 +79,8 @@ type PackOption struct {
 	Backend Backend
 	// Timeout cancels execution once exceed the specified time.
 	Timeout *time.Duration
+	// EncryptRecipients holds keys to converting OCI tar(.gz) blob to nydus encrypted blob.
```

Comment:
keys for ?

Reply:
These key are the same with the keys which used to encrypt Nydus bootstrap with ocicrypt.
I decided to change this item to 'Encrypt' which is a bool value, because the  useage of these specific keys in the future is not very likely.